File: sandbox/dotnet/src/Dropbox.SignSandbox/TemplateRemoveUserExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TemplateRemoveUserExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        var templateRemoveUserRequest = new TemplateRemoveUserRequest(
            emailAddress: "george@dropboxsign.com"
        );

        try
        {
            var response = new TemplateApi(config).TemplateRemoveUser(
                templateId: "f57db65d3f933b5316d398057a36176831451a35",
                templateRemoveUserRequest: templateRemoveUserRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling TemplateApi#TemplateRemoveUser: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: sandbox/node/src/TemplateRemoveUserExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TemplateApi();
apiCaller.username = "YOUR_API_KEY";
// apiCaller.accessToken = "YOUR_ACCESS_TOKEN";

const templateRemoveUserRequest: models.TemplateRemoveUserRequest = {
  emailAddress: "george@dropboxsign.com",
};

apiCaller.templateRemoveUser(
  "f57db65d3f933b5316d398057a36176831451a35", // templateId
  templateRemoveUserRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling TemplateApi#templateRemoveUser:");
  console.log(error.body);
});
